src/style/global.sass

```diff
@@ -31,3 +32,50 @@ ul, ol
     padding-inline: 2rem
     width: 100%
     box-sizing: border-box
+
+.centered-container 
+    display:flex
+    width: 100%
+    justify-content:center
+    align-items:center
+    
+/* classes for buttons (jonathan) */
+.button 
+  padding-top: 0.85em
+  padding-bottom: 0.85em
+  padding-right: 2em
+  padding-left: 2em
+  font-weight: bold
+  border: 1px solid transparent
+  border-radius: 1rem
+  cursor: pointer
+
+.primary-button 
+    background-color: $color-primary
+
+.primary-button:hover
+    background-color: transparent
+    border: 1px solid $color-primary
+
+.secondary-button
+    background-color: transparent
+    border: 1px solid $color-primary
+
+.secondary-button:hover
+    background-color: $color-primary
+
+.super-header 
+    color: #17E6A8
```

Comment:
Use the color as a variable `$color-primary-light`, so it change if we change the global colors.